airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/assets.json

```
@@ -0,0 +1,30 @@
+{
+  "consumingDags": "消费 DAGs",
+  "createEvent": {
+    "button": "创建事件",
+    "manual": {
+      "description": "手动创建资源事件",
+      "extra": "资源事件额外信息",
+      "label": "手动"
+    },
+    "materialize": {
+      "description": "触发此资源上游的 DAG",
```

```suggestion
      "description": "触发此资源上游的 Dag",
```

Based on the recent discussion in the email thread, `DAG` is replaced by `Dag` or `dag`. Example PR：https://github.com/apache/airflow/pull/55099 

It applied to all other places using `DAG`.